CHANGELOG.md

```diff
@@ -7,22 +7,26 @@
 This release contains major changes to the web layer, including TCP and HTTP handling as well as all web endpoint handlers. Due to these changes, some functionality was retired and Relay responds differently in specific cases.
 
 Configuration:
+
```

Comment:
These blank lines have been added by the formatter, so I thought about leaving them in place to avoid saving without formatting every time there's a change in the changelog.